src/Layers/xrRender/blenders/dx11RainBlender.cpp

```diff
@@ -5,6 +5,9 @@ void CBlender_rain::Compile(CBlender_Compile& C)
 {
     IBlender::Compile(C);
 
+    auto const PreviousSmapSize = RImplementation.o.smapsize;
+    RImplementation.o.smapsize = RImplementation.o.rain_smapsize;
```

Comment:
`RImplementation.o` should never be changed, so a third run-time variable `RImplementation.m_CurrentSMAPSize` should be introduced